build.sbt

```diff
@@ -0,0 +1,22 @@
+lazy val `sbt-circe-io` =
+  project.in(file(".")).enablePlugins(NoPublishPlugin).aggregate(core)
+
+lazy val docs = project
+  .enablePlugins(CirceIoSitePlugin)
```

Comment:
```suggestion
lazy val docs = project
  .in(file("site"))
  .enablePlugins(CirceIoSitePlugin)
```

Reply:
This bit I do not understand. We have no folder named `site` in the root of the project. What magic does the plugin do? 😸

Reply:
It just needs a place to put the `target/` directories where the site is generated in. They can't go in the `docs` folder, for Reasons:tm:.